### ISSUE TITLE
Example of fine-tuning DiT for layout analysis

### ISSUE THREAD

**Issue by maiiabocharova** [NielsRogge/Transformers-Tutorials#108]
Hello, thank you a lot for amazing tutorials!

Can you please also do an example on how to fine-tune DiT on custom dataset for layout analysis? Or should it be roughly the same as DetrForObjectDetection, only changing the name of the model?

Best

**Comment by pashanitw** [comment]
yes i am also looking for fine tuning DiT. its impossible to understand the code.

1) my understanding is Dit should be backbone
2) the output of the DiT should be passed to the object detection model 

is that correct?
